source/documentation/syntax/comments.liquid

```diff
@@ -0,0 +1,140 @@
+---
+title: Comments
+introduction: >
+  The way Sass comments work differs substantially between SCSS and the indented
+  syntax. Both syntaxes support two types of comments: comments defined using
+  `/* */` that are (usually) compiled to CSS, and comments defined using `//` that
+  are not.
+---
+{% markdown %}
+## In SCSS
+
+Comments in SCSS work similarly to comments in other languages like JavaScript.
+**Single-line comments** start with `//`, and go until the end of the line.
+Nothing in a single-line comment is emitted as CSS; as far as Sass is concerned,
+they may as well not exist. They're also called **silent comments**, because
+they don't produce any CSS.
+
+**Multi-line comments** start with `/*` and end at the next `*/`. If a
+multi-line comment is written somewhere that a [statement][] is allowed, it's
+compiled to a CSS comment. They're also called **loud comment**, by contrast
+with silent comments. A multi-line comment that's compiled to CSS may contain
+[interpolation][], which will be evaluated before the comment is compiled.
+
+By default, multi-line comments be stripped from the compiled CSS in [compressed
+mode][]. If a comment begins with `/*!`, though, it will always be included in
+the CSS output.
+
+[statement]: structure#statements
```

Comment:
currently points to:
/documentation/syntax/comments/structure#statements

but I think we want:
/documentation/syntax/structure#statements